cmd/rancher_release/check_rancher_rc_deps.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"errors"
+
+	"github.com/rancher/ecm-distro-tools/release/rancher"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+func checkRancherRCDepsCommand() *cli.Command {
+	return &cli.Command{
+		Name:  "check-rancher-rc-deps",
+		Usage: "check if the Rancher version specified by the commit or pre-release title does not contain development dependencies and rc tags",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:     "commit",
+				Aliases:  []string{"c"},
+				Usage:    "last commit for a final rc",
+				Required: true,
+			},
+			&cli.StringFlag{
+				Name:     "org",
+				Aliases:  []string{"o"},
+				Usage:    "organization name",
+				Required: false,
+			},
+			&cli.StringFlag{
+				Name:     "repo",
+				Aliases:  []string{"r"},
+				Usage:    "rancher repository",
+				Required: false,
+			},
+			&cli.StringFlag{
```

Comment:
The files required by this program are coded in. Is the files parameter required? Maybe it adds something useful, but it seems like an extra hoop that could invite user error.

Reply:
Yes, as requested I changed the logic to run firstly locally and if needed, remote by commit hash